### PR TITLE
1609149 - Remove `ping_type` from pings.

### PR DESCRIPTION
**Pull request by badboy**
Needs https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/485 to land first.

### CODE REVIEW

File: glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt [hunk 1]
```diff
@@ -272,8 +272,6 @@ class MetricsPingSchedulerTest {
 
             // Validate the received data.
             checkPingSchema(metricsJson)
-            assertEquals("The received ping must be a 'metrics' ping",
```
Comment (Dexterp37) [inline glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt]: Can we retain the check and use the submission URL here and at the other locations?

**Comment by badboy** [comment]
> The schema change was merged. Does it need to be deployed as well? Is it already live?

I'll check if it is live with the team. This only becomes relevant once this lands in a release used by applications though.

I'm adding a changelog entry, so we don't forget.

File: glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt (hunk 1 above)
Comment (badboy) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt]: adding it back

**Comment by badboy** [comment]
I added back the checks where it makes sense.
In some parts we were already checking against the URL (in the helper functions), so we can avoid the additional check there.

**Comment by badboy** [comment]
```
java.lang.AssertionError: Task queue is cleared expected:<0> but was:<3>
```

but only for `testReleaseUnitTest`.
Might be a race condition due to the recent changes? Need to look into that further first.

**Comment by Dexterp37** [comment]
> Might be a race condition due to the recent changes? Need to look into that further first.

Maybe due to my async flush?

**Comment by badboy** [comment]
Waiting for reason codes/baseline PRs to land and a release first.

**Pull request merged**